packages/model/Project.ts

```diff
@@ -22,19 +22,33 @@ export interface Project extends RdfResource {
   creator: Person
   label: string
   jobCollection: Link<JobCollection>
-  cubeIdentifier: string
   maintainer: Link<Organization>
   publishedRevision: number
+  sourceKind: NamedNode
+}
+
+export interface CsvProject extends Project {
+  cubeIdentifier: string
+}
+
+export interface ImportProject extends Project {
+  importCube: NamedNode
+  importFromGraph: NamedNode | undefined
+  importFromEndpoint: NamedNode
 }
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface ProjectsCollection extends Collection<Project> {
+export interface ProjectsCollection extends Collection<CsvProject | ImportProject> {
 
 }
 
+export const isCsvProject = (project: CsvProject | ImportProject): project is CsvProject => {
+  return project.pointer.out(dcterms.identifier).terms.length > 0
```

Comment:
Maybe this should instead check the `projectSourceKind`?

Reply:
That would make sense

Reply:
Project doesn't seem to have a `cc:projectSourceKind` in the frontend. I guess we could also check if the project has a `csvMapping`?

Reply:
Is this about existing project? We did not persist that property until now. Make sure you run the SPARQL from the PR description

Reply:
Ah that's probably it then. Will do that, thanks